pkg/terminal/terminal.go

```diff
@@ -101,19 +101,18 @@ func New(client service.Client, conf *config.Config) *Term {
 	}
 
 	if conf.SourceListLineColor < ansiBlack ||
-	conf.SourceListLineColor > ansiBrWhite {
-		fmt.Fprintf(os.Stderr, "invalid source-list-line-color, defaulting to ansiBlue(34)\n")
```

Comment:
Removed fmt.Fprintf() per review comments
Range check against lower/upper valid ansi color values was left, as only testing against 0 will handle user leaving the pref commented out in config.yml, but not a user setting out of range values, eg

source-list-line-color: -1

yields list output
...
[-1m    22:		MOVQ	$0, R10
[-1m    23:		MOVQ	$0, R8
...